res/values/strings.xml

```diff
@@ -1241,6 +1242,7 @@
     <string name="conversation__menu_leave_group">Leave group</string>
     <string name="conversation__menu_view_all_media">All media</string>
     <string name="conversation__menu_conversation_settings">Conversation settings</string>
+    <string name="conversation__menu_pinned_messages">View all pins</string>
```

Comment:
Not important but maybe call this "View pinned messages", pin is usually the abbreviation for pin number/code and not pinned message.

Reply:
I agree